src/internal/styles/forms/mixins.scss

```diff
@@ -147,13 +147,26 @@
     @include utils.base-pseudo-element;
     box-shadow: awsui.$shadow-container;
     border-radius: awsui.$border-radius-container;
-    mix-blend-mode: multiply;
+  }
+}
+
+@mixin container-border {
+  border-radius: awsui.$border-radius-container;
+  box-sizing: border-box;
+
+  &::before {
+    @include utils.base-pseudo-element;
+    // Reset border color to prevent it from flashing black during card selection animation
+    border-color: transparent;
+    border-top: awsui.$border-container-top-width solid awsui.$color-border-container-top;
+    border-radius: awsui.$border-radius-container;
+    z-index: 1;
   }
 
-  // HACK: because of the issues with IE's box shadow rendering we draw
-  // a border and don't mind the pixel fitting
-  @media screen and (-ms-high-contrast: active) {
-    border: constants.$control-border-width solid awsui.$color-border-container-top;
+  &::after {
+    @include utils.base-pseudo-element;
+    outline: 1px solid awsui.$color-border-divider-default;
```

Comment:
The `@container-shadow` mixin is intended to provide the style definition for containers, whether that's via shadow, border, or both. 
Since we removed the box shadow definitions in `style-dictionary/visual-refresh/shadows.ts`, and this seems to be the only line that is different between these two mixins, can we create/use a token for this border instead of creating a whole new mixin for just the border? That way it can be used for both classic and VR without needing to add the `isVisualRefresh` hook in downstream components like container, cards, modal, etc. We can rename the mixin to something more generic like @container-style or @container-outline if that helps.